### ISSUE TITLE
Incorrect name for A2 bomb rack

### ISSUE THREAD

**Issue by alistair3149** [StarCitizenWiki/API#70]
A2 bomb rack (`BMBRCK_CRUS_STARL_Left`) has the name `MSD-313 Missile Rack`.
https://github.com/StarCitizenWiki/scunpacked-data/blob/3cfde6041ea161d09b721ef42e278598468c0b2a/v2/items/bmbrck_crus_starl_left.json#L4

**Comment by octfx** [comment]
Isn't that the current output on the API?
![image](https://github.com/StarCitizenWiki/API/assets/6594492/2e55b2c7-2670-4ea6-823b-444137dffa87)

**Comment by alistair3149** [comment]
Yeah, I'm wondering if that's an issue with how SCUnpacked is parsing the data since there's no way it is the msd-313. How did it get the localization info? https://github.com/StarCitizenWiki/scunpacked-data/blob/3cfde6041ea161d09b721ef42e278598468c0b2a/v2/items/bmbrck_crus_starl_left-raw.json#L160

**Comment by octfx** [comment]
Ah, got it!
Looking at the game data the A2 indeed has a MSD-313 Missile Rack equipped
![image](https://github.com/StarCitizenWiki/API/assets/6594492/bc9762d7-afff-415f-8d99-1781c387aca0)
![image](https://github.com/StarCitizenWiki/API/assets/6594492/e918e02d-aa68-4c15-8fdb-2cea82d42186)

**Comment by alistair3149** [comment]
That is so weird. I guess both Erkul and SPV ignore the localization string for that then

**Comment by octfx** [comment]
<img width="793" height="414" alt="Image" src="https://github.com/user-attachments/assets/42526053-973f-4dff-a7b7-2a2ce6a699ce" />
Game data seems to be fixed
